<launch>
 
  <param name="kinect_head/driver/image_mode" value="1" />
  
  <include file="$(find openni_launch)/launch/openni.launch">
    <arg name="load_driver" value="true" />
    <arg name="depth_registration" value="true" />
  </include>

  <include file="$(find ml_classifiers)/launch/classifier_server.launch" />

  <arg name="2d_recognition_data" default="_" /> <!-- leave '_' in here-. roslaunch fucks up the params otherwise-->
  
  <node pkg="suturo_perception_rosnode" name="suturo_perception_rosnode" type="suturo_perception_rosnode" output="screen" args="$(arg 2d_recognition_data)"></node>
  <param name="suturo_perception/point_topic" type="string" value="/kinect_head/depth_registered/points" />
  <param name="suturo_perception/color_topic" type="string" value="/kinect_head/rgb/image_color" />
  <param name="suturo_perception/frame_id" type="string" value="/head_mount_kinect_rgb_optical_frame" />

</launch>
